tests/unit/explore_test.sh

```diff
@@ -236,6 +257,20 @@ function test_parse_explore_options()
   assertEquals "($LINENO)" '0' "$ret"
   assertEquals "($LINENO)" 'HEADER' "${options_values['SCOPE']}"
 
+  unset options_values
+  declare -gA options_values
+  parse_explore_options -C
+  ret="$?"
+  assertEquals "($LINENO)" '0' "$ret"
```

Comment:
Use `(${LINENO})` in all of the assertions, and you don't need quotes around numbers.

Reply:
I think this change would be out of place for this PR. Would it be a good idea to change this in another commit?

Reply:
I think Siqueira was alluding to changing all occurrences in the additions of this PR as we are concatenating the parenthesis with the value of `LINENO`. The pattern in this file is misleading, so don't use it as basis :smile: